dev-libs/antlr-c/antlr-c-3.5.2-r2.ebuild

```diff
@@ -42,24 +39,20 @@ multilib_src_configure() {
 		*) econfargs+=( --disable-64bit ) ;;
 	esac
 
-	econf "${econfargs[@]}"
+	CONFIG_SHELL="${BROOT}"/bin/bash econf "${econfargs[@]}"
 }
 
-src_compile() {
-	multilib-minimal_src_compile
-
-	if use doc; then
+src_install() {
+	if use doc ; then
```

Comment:
I would've kept this in `src_compile`, but w/e

Reply:
Fixed.